### PR TITLE
Create ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58122](https://cumulocity.atlassian.net/browse/MTM-58122)
Original PR: [6759](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6759)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: SmartRule_pin_dropdown_to_the_card_body (#6553) [GRAFT][release/cd] (#6759)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Smart rule pin dropdown moved to the card body
```

File: content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58122
+version: 1020.9.1
+---
+SmartRule_pin_dropdown_to_the_card_body (#6553) [GRAFT][release/cd] (#6759)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration interface. The new placement of the pin dropdown is expected to enhance usability and reduce confusion for users when setting up and managing their smart rules.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@darluk Please check the suggestions in this ticket and make any required edits.

**Review by darluk — APPROVED** [review]

**Review by pmich-sag — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58122
+version: 1020.9.1
+---
+To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration interface. The new placement of the pin dropdown is expected to enhance usability and reduce confusion for users when setting up and managing their smart rules.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md:17]:
```suggestion
To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration. 
```

File: content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Smart rule pin dropdown moved to the card body
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md:3]:
```suggestion
title: Smart rule pin dropdown moved from the header to the card body
```

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md:3]:
```suggestion
title: Smart rule pin dropdown moved from the header to the card body
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**